stripe/_list_object.py

```diff
@@ -190,13 +217,15 @@ def next_page(
         params_with_filters.update({"starting_after": last_id})
         params_with_filters.update(params)
 
-        result = self.list(
-            api_key=api_key,
-            stripe_version=stripe_version,
-            stripe_account=stripe_account,
-            **params_with_filters,
-        )
-        return result
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", category=DeprecationWarning)
+            result = self.list(  # pyright: ignore[reportDeprecated]
```

Comment:
What are we going to do here instead?

Reply:
Rename it to `_list`, but in [Address review](https://github.com/stripe/stripe-python/pull/1101/commits/cc8e237402eb70696bad6cd98ecb21e2093db4db) I went ahead and just defined `_list` already.